### PR TITLE
Decoupling TPC digitizer workflow from reco workflow

### PR DESCRIPTION
**Pull request by matthiasrichter**
Workflows can be combined on the command line by using pipe. Removing
the early implementation of combining the two workflows.

Example usage:
```
o2-sim-digitizer-workflow --onlyDet TPC | o2-tpc-reco-workflow --input-type digitizer --tpc-lanes 4
```

### CODE REVIEW

File: Steer/DigitizerWorkflow/CMakeLists.txt [hunk 1]
```diff
@@ -46,11 +46,10 @@ o2_add_executable(digitizer-workflow
                                         O2::PHOSSimulation
                                         O2::CPVSimulation
                                         O2::TOFSimulation
-					                    O2::TOFCalibration
+                                        O2::TOFCalibration
                                         O2::TOFReconstruction
                                         O2::TOFWorkflowUtils
                                         O2::TPCSimulation
-                                        O2::TPCWorkflow
```
Comment (matthiasrichter) [inline Steer/DigitizerWorkflow/CMakeLists.txt:53]: @sawenzel we can remove the dependency now, but we can also leave it if it's planned to move the TPC specs to `O2::TPCWorkflow` anyhow. Depending on the plans for cleaning up the digitizer workflow.

Comment (sawenzel) [reply Steer/DigitizerWorkflow/CMakeLists.txt:53]: Yes, I think we should move these things into the proper place. So let's leave it.

**Comment by sawenzel** [comment]
For my understanding. In the example
```
o2-sim-digitizer-workflow --onlyDet TPC | o2-tpc-reco-workflow --input-type digitizer --tpc-lanes 4
```
will `--tpc-lanes` also act on the digitizer?

**Comment by ktf** [comment]
@sawenzel IIRC, it should apply to whatever declares it as a workflow option.

**Comment by sawenzel** [comment]
Ok. It was not evident to me that it applies to the left of the `|` operator as well.

**Comment by ktf** [comment]
The problem is that in order to guarantee stability of the topological sort across all the merged sub-workflows, the workflow options (which, as a reminder, affect topology) need to be consistent all across the board.

**Pull request merged**